src/main/webapp/admin_dashboard.html

```diff
@@ -88,5 +88,19 @@ <h2> Last 10 User Tasks</h2>
         </div>
       </main>
     </div>
+    <div id="popup-overlay">
+		<div class="popup">
+			<h2>Edit Info for Task</h2>
+			<form action="">
+				<input type="text" placeholder="Task Details" id="task-details">
+
+				<input type="text" placeholder="Category" id="category">
+				<input type="submit" value="Submit">
+			</form>
+			<a href="#" class="popup-close">&times;</a>
+		</div>
+	</div>
+	
+	<h1>Click <a href="#popup-overlay">here</a> to open the popup box</h1>
```

Comment:
For a future PR: hyperlinks with just "here" as the inner text are confusing when read aloud to users by a screen reader; we should aim to make the text more descriptive of what the link does.